reactor-core/src/main/java/reactor/core/publisher/FluxTimeout.java

```diff
@@ -87,13 +99,19 @@ public void subscribe(CoreSubscriber<? super T> actual) {
 		source.subscribe(main);
 	}
 
+	private String resolveName(Publisher<?> source) {
+		Tuple2<String, List<Tag>> nameAndTags = FluxMetrics.resolveNameAndTags(source);
```

Comment:
To avoid the warning, you could check the source is `instanceOf Scannable` and do nothing if it is not.
But I'd also avoid using the default `name` (which was critical in Micrometer, but not that much here).
So maybe copying the code over from FluxMetrics makes sense 🤔